ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -188,50 +188,47 @@ StatusSectionLayout {
         return root.chatContentModule.chatDetails.isUsersListAvailable
     }
 
-    rightPanel: Component {
-        id: userListComponent
-        UserListPanel {
-            readonly property var usersStore: ChatStores.UsersStore {
-                usersModule: !!root.chatContentModule ? root.chatContentModule.usersModule : null
-                chatDetails: !!root.chatContentModule ? root.chatContentModule.chatDetails : null
-                chatCommunitySectionModule: root.rootStore.chatCommunitySectionModule
-            }
+    rightPanel: UserListPanel {
```

Comment:
This will confict with some recent changes I pushed I think!

Reply:
that's fine. All I'm interested in is to drop the `Component` and use `UserListPanel` directly. I've just copied whatever is on master in `UserListPanel` body